webrtc-stats.html

```diff
@@ -2206,6 +2206,16 @@ <h2>
                   Only valid for video. The current reason for limiting the resolution and/or
                   framerate, or "none" if not limited.
                 </p>
+                <p>
+                  If there are multiple limitation reasons at the same time, the
+                  implementation is encouraged to attempt to report the most
+                  limiting reason. If the implementation does not know how to
+                  determine this it SHOULD prefer to report reasons in the
+                  following order of priority: "bandwidth", "cpu", "other". In
+                  other words, an implementation that is able to tell we are
+                  both "bandwidth" and "cpu" limited but cannot tell which
+                  factor is the most limiting factor will report "bandwidth".
```

Comment:
First thing, SHOULD always follow when not to apply the preference order. In this case the sentence is trying to say MUST use this order if the endpoint does not know.

Secondly, we are not explaining the rationale why this preference order is chosen as the default.

Reply:
I rephrased things. But because what the "most limiting factor" is, even under ideal circumstances: a guestimate - I couldn't faithfully include the word "MUST". I used "SHOULD" in both places. The changes are uploaded in a new PR: #526. Closing this one.